dvc/scm/git/__init__.py

```diff
@@ -235,3 +235,52 @@ def belongs_to_scm(self, path):
 
     def get_tree(self, rev):
         return GitTree(self.git, rev)
+
+    def _get_diff_trees(self, a_ref, **kwargs):
+
+        from gitdb.exc import BadObject, BadName
+
+        trees = {"a_tree": None, "b_tree": None}
+        commits = []
+        try:
+            if "b_ref" in kwargs and kwargs["b_ref"] is not None:
+                b_ref = kwargs["b_ref"]
+                a_commit = self.git.commit(a_ref)
+                b_commit = self.git.commit(b_ref)
+                commits.append(str(a_commit))
+                commits.append(str(b_commit))
+                # other way of getting the diff between commits :
```

Comment:
Is that other way better? I'm not sure how should I interpret this.

Reply:
I think, that method is not applicable since we switched to not passing around strings :)